tests/dummy/app/services/fm-config.js

```diff
@@ -0,0 +1,13 @@
+import FormMaster2000ConfigService from 'ember-form-master-2000/services/fm-config';
+
+export default FormMaster2000ConfigService.extend({
+
+  // default config
```

Comment:
Just want to flesh this out a bit so its crystal clear. Can you change to something like the following. 
```
By default, form master uses the default classes for Bootstrap 3 style forms.  To
change this, extend the `fm-config` service and override the class names that you want.
For a list of the default classes see the base `fm-config` service in:

https://github.com/Emerson/ember-form-master-2000/blob/master/addon/services/fm-config.js

For example for semantic-ui style inputs you would include the following overrides:
formClass: 'ui form',
wrapperClass: 'field',
errorClass: 'error'
```